### ISSUE TITLE
Hunter. 'Call Pet'

### ISSUE THREAD

**Issue by czeshirecat** [brittyazel/Neuron#372]
Dragging 'Call Pet' (or it's sub options) to neuron bar doesn't work in oh so many ways :D 
(Love the addon, ty)

**Comment by brittyazel** [comment]
Can you please specify in which ways it does not work? And is it specific to Call Pet? What about Pet Utility?

My only hunter toon is a level 10 and only has a single pet so my testing capacity is limited. Call Pet works perfectly fine with my single pet, for what it's worth.

**Comment by teh-fixerer** [comment]
For me, I found that dragging the call pet flyout seems to perform ok.   However If I try to drag one of the pets out from the spell book flyout, the macro neuron makes is /cast call "petname", rather than /cast call pet(1) etc.

**Comment by brittyazel** [comment]
Yeah, it's doing what it 'should', i.e. what works for 99.99% of all other spells in the game which is to call by name. I need to override this in the case of this particular situation though since it appears the game wants it in /cast call pet(#) form.

So, _that_, issue if fixable. What about other issues? Is drag pet leading to Lua errors or anything of the sort @czeshirecat? You somewhat left me hanging with "doesn't work in oh so many ways". (It's usually better when requesting help to list out the 'so many' ways lol)
